<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyLSS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_11">
    <item row="1" column="1" colspan="2">
     <widget class="QGroupBox" name="plotterBox">
      <property name="minimumSize">
       <size>
        <width>680</width>
        <height>350</height>
       </size>
      </property>
      <property name="title">
       <string>Chromatogram</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="3">
     <widget class="QGroupBox" name="settingBox">
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Settings</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QToolBox" name="toolBox">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>240</width>
            <height>549</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Data input</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <widget class="QPushButton" name="addButton">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="removeButton">
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QListView" name="listView">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>240</width>
            <height>549</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Gradient Parameters</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="7" column="0">
            <widget class="QDoubleSpinBox" name="doubleSpinBox_6">
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="singleStep">
              <double>0.050000000000000</double>
             </property>
             <property name="value">
              <double>0.969000000000000</double>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="2">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>cm</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>mm</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Âµm&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Column length</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Particle size</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QDoubleSpinBox" name="ColumnPorositySpinBox">
               <property name="decimals">
                <number>2</number>
               </property>
               <property name="value">
                <double>1.700000000000000</double>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="ColumnLenghtSpinBox">
               <property name="decimals">
                <number>0</number>
               </property>
               <property name="value">
                <double>15.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Column diameter</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="CulumnDiameterSpinBox">
               <property name="value">
                <double>2.100000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="1" column="0">
              <widget class="QComboBox" name="modelBox"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Select LSS model..</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="4" column="0">
            <layout class="QGridLayout" name="gridLayout_9">
             <item row="0" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Gradient time</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
               <property name="decimals">
                <number>2</number>
               </property>
               <property name="minimum">
                <double>1.000000000000000</double>
               </property>
               <property name="maximum">
                <double>200.000000000000000</double>
               </property>
               <property name="value">
                <double>10.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="8" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Dwell volume</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="doubleSpinBox_5">
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="minimum">
                <double>0.001000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.050000000000000</double>
               </property>
               <property name="value">
                <double>0.370000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Gradient Start</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
               <property name="value">
                <double>5.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Gradient End</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
               <property name="value">
                <double>95.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Flow Rate</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="doubleSpinBox_1">
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="minimum">
                <double>0.001000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.050000000000000</double>
               </property>
               <property name="value">
                <double>0.300000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>t0</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QGroupBox" name="plotterBox_2">
      <property name="minimumSize">
       <size>
        <width>680</width>
        <height>150</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>250</height>
       </size>
      </property>
      <property name="title">
       <string>Data</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="0">
        <widget class="QTableView" name="tableView"/>
       </item>
       <item row="0" column="1">
        <widget class="QPlainTextEdit" name="plainTextEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>9999999</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="plainText">
          <string>Critical resolution: 1.80
Molecules: 
  - A
  - B</string>
         </property>
         <property name="overwriteMode">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="title">
       <string>Plot options</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="QPushButton" name="zoomButton">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>Zoom</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="panButton">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>Pan</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="rescaleButton">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>Rescale</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="exportchromatogramButton">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>999999</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>Export chromatogram</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="setplotoptionsButton">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>9999999</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>Set plot options</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>990</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuCalculation">
    <property name="title">
     <string>Calculation</string>
    </property>
    <addaction name="actionCalcLSSParameter"/>
    <addaction name="separator"/>
    <addaction name="actionAutomaticElutionWindowStretching"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDocumentation"/>
    <addaction name="actionTecnical_Support"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuPlot_2">
    <property name="title">
     <string>Plot</string>
    </property>
    <addaction name="actionSelectivityMap"/>
    <addaction name="actionResolutionMap"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCalculation"/>
   <addaction name="menuPlot_2"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCalcLSSParameter">
   <property name="text">
    <string>Calculate LSS parameter</string>
   </property>
  </action>
  <action name="actionAutomaticElutionWindowStretching">
   <property name="text">
    <string>Automatic elution window stretching</string>
   </property>
  </action>
  <action name="actionGetBestIsocraticConditions">
   <property name="text">
    <string>Get best isocratic conditions</string>
   </property>
  </action>
  <action name="actionOpen_Session">
   <property name="text">
    <string>Open Session</string>
   </property>
  </action>
  <action name="actionSave_Session">
   <property name="text">
    <string>Save Session</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionTecnical_Support">
   <property name="text">
    <string>Tecnical Support</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionExport_LSS_Parameter">
   <property name="text">
    <string>Export LSS Parameter</string>
   </property>
  </action>
  <action name="actionSelectivityMap">
   <property name="text">
    <string>Selectivity Map</string>
   </property>
  </action>
  <action name="actionResolutionMap">
   <property name="text">
    <string>Resolution Map</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
